docs/root/source/data-models/block-model.rst

```diff
@@ -20,7 +20,7 @@ A block has the following structure:
 - ``id``: The hash of the serialized ``block`` (i.e. the ``timestamp``, ``transactions``, ``node_pubkey``, and ``voters``). This is also a database primary key; that's how we ensure that all blocks are unique.
 
 - ``block``:
-    - ``timestamp``: The Unix time when the block was created. It's provided by the node that created the block. See `the page about timestamps <https://docs.bigchaindb.com/en/latest/timestamps.html>`_.
+    - ``timestamp``: The Unix time when the block was created. It's provided by the node that created the block.
```

Comment:
Cool, in agreement with this, I expected there would be some discussion about this page, but as in my other comment it would be helpful to me to re-state the objectives of the timestamps page, as it seemed to deal primarily with the ambiguity arising from the user provided timestamp.